src/Components/Main/Pages/ProfilePage/ProfileStatus/ProfileStatus.js

```diff
@@ -0,0 +1,49 @@
+import classes from './ProfileStatus.module.scss';
+import { useEffect, useState } from 'react';
+import { Preloader } from '../../../../Common/Preloader/Preloader';
+
+export const ProfileStatus = ({status, updateStatus, isFetchingStatus}) => {
+  const [editMode, setEditMode] = useState(false);
+  const [localStatus, setLocalStatus] = useState(status);
+
+  useEffect(() => {
+    setLocalStatus(status);
+  }, [status]);
+
+  const onActivateEditMode = () => {
+    setEditMode(true);
+  };
+
+  const onDeactivateEditMode = () => {
+    setEditMode(false);
+    updateStatus(localStatus);
+  };
+
+  const onChangeLocalStatus = (event) => {
+    setLocalStatus(event.target.value);
+  }
+
+  return (
+    <div className={classes.statusBlock}>
+      {
+        editMode ?
+          <div className={classes.status}>
+            <b className={classes.title}>Status:</b>
+            <textarea className={classes.inputStatus}
+                      onChange={onChangeLocalStatus}
+                      onBlur={onDeactivateEditMode}
+                      autoFocus={true}
+                      value={localStatus} />
+          </div> :
+          <div className={classes.status}>
+            <b className={classes.title}>Status:</b>
+            {
+              isFetchingStatus ?
+                <Preloader className={classes.statusPreloader} /> :
+                <p className={classes.statusText} onDoubleClick={onActivateEditMode}>{status || 'no status'}</p>
+            }
+          </div>
+      }
+    </div>
+  );
```

Comment:
As I see your have
```
<div className={classes.status}>
            <b className={classes.title}>Status:</b>
```
in both conditions, I can propose the change: 
```
return (
    <div className={classes.statusBlock}>
      <div className={classes.status}>
        <b className={classes.title}>Status:</b>
        {editMode ? (
          <textarea
            autoFocus
            className={classes.inputStatus}
            onBlur={onDeactivateEditMode}
            onChange={onChangeLocalStatus}
            value={localStatus}
          />
        ) : (
          (isFetchingStatus && <Preloader className={classes.statusPreloader} />) || (
            <p className={classes.statusText} onDoubleClick={onActivateEditMode}>
              {status || 'no status'}
            </p>
          )
        )}
      </div>
    </div>
  );
```

Reply:
Added changes, but used a different code structure!